charts/external-es-proxy/templates/_helpers.tpl

```diff
@@ -101,3 +101,15 @@ imagePullSecrets:
 {{ .Values.images.awsproxy.repository }}:{{ .Values.images.awsproxy.tag }}
 {{- end }}
 {{- end }}
+
+
+
```

Comment:
The space between blocks keeps getting longer and longer. It's 1 line in some places, 2 lines in others, but I think 3 lines is a bit too much.
```suggestion

```